tests/python/pants_test/pants_run_integration_test.py

```diff
@@ -284,6 +299,13 @@ def run_pants_with_workdir_without_waiting(self, command, workdir, config=None,
                    '--cache-bootstrap-read', '--cache-bootstrap-write'
                    ])
 
+    if self.should_configure_pantsd():
+      args.append("--enable-pantsd=True")
+      args.append("--no-shutdown-pantsd-after-run")
```

Comment:
I thought that our intent right now was to actually enable `--shutdown-pantsd-after-run`... but I think that you also determined that because of the temporary workdir, pantsd will shutdown after each run anyway?

Reply:
So we don't want to shutdown after every run _inside_ the tests, because some of them rely on pantsd living run after run (although they don't rely on this codepath). We do want to set it (probably) in the overpants, the test coordinator, via travis env var. But this should be done at the same time as we enable pantsd by default, and shouldn't affect these changes at all.

Reply:
I would argue that those tests should be marked hermetic then. But this is ok too!